sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -2842,13 +2844,16 @@ public void testGroupByWithSelectAndOrderByProjections()
   @Test
   public void testTopNWithSelectProjections()
   {
+    final Map<String, Object> context =
```

Comment:
i think this is repeated enough across files to be worth pushing down as one of the static contexts